packages/spright-components/src/chat/conversation/styles.ts

```diff
@@ -18,8 +23,22 @@ export const styles = css`
         flex-direction: column;
         justify-content: flex-start;
         row-gap: 32px;
-        padding: ${mediumPadding};
-        background: ${applicationBackgroundColor};
+        padding: ${mediumPadding} ${standardPadding} ${mediumPadding}
+            ${standardPadding};
+        border: ${borderWidth} solid ${applicationBackgroundColor};
+        background: linear-gradient(
+            ${chatConversationBackgroundGradientStartColor},
+            ${chatConversationBackgroundGradientEndColor}
+        );
```

Comment:
What if we added a [gradient](https://developer.mozilla.org/en-US/docs/Web/CSS/gradient) or even [image](https://developer.mozilla.org/en-US/docs/Web/CSS/image) as a new [type suffix](https://github.com/ni/nimble/blob/main/packages/nimble-components/src/theme-provider/design-token-names.ts#L287)? (think I'd lean more to `image`)

Can still design the token to be responsive to theme and just avoid the intermediary tokens (which don't seem valuable on their own?).

Reply:
Nice idea, I went with "image" as the name for the new type. It was pretty easy to plumb it through and render it in the theme-aware tokens story (though it's hard to see the gradient at such a small size).

I ended up using `string` as the type. There are [some native types](https://developer.mozilla.org/en-US/docs/Web/API/CSSImageValue) to represent this concept but it didn't look like I could create an instance of a linear gradient object in that type system so we pass around CSS strings like we do on some of our other token types.